### PR TITLE
Create ui-c8y-1021-0-0-added-types-to-widget-configuration.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by eniosultan

Ticket: [MTM-38876](https://cumulocity.atlassian.net/browse/MTM-38876)
Original PR: [6903](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/6903)

[MTM-38876]: https://cumulocity.atlassian.net/browse/MTM-38876?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-added-types-to-widget-configuration.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Added types to widget configuration
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-added-types-to-widget-configuration.md]: The Claude AI suggested to use the following title:
```suggestion
title: Add type information to widget configuration
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-added-types-to-widget-configuration.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-38876
+version: 1021.0.0
+---
+Added types to widget configuration
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-added-types-to-widget-configuration.md]: The Claude AI suggested to use the following body:
```suggestion
To provide more clarity and improve validation when configuring widgets, type information has been added to the widget configuration. Previously, the widget configuration did not include any type information, which could lead to confusion and misconfiguration. With this change, each configuration property now specifies its expected type, such as string, number, or boolean. This enhancement helps users to correctly configure widgets and catch potential errors early in the configuration process, leading to a smoother widget setup experience.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@eniosultan Please check this PR.

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-added-types-to-widget-configuration.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Add type information to widget configuration
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-added-types-to-widget-configuration.md]:
```suggestion
title: Added type information to widget configuration
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-added-types-to-widget-configuration.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-38876
+version: 1021.0.0
+---
+To provide more clarity and improve validation when configuring widgets, type information has been added to the widget configuration. Previously, the widget configuration did not include any type information, which could lead to confusion and misconfiguration. With this change, each configuration property now specifies its expected type, such as string, number, or boolean. This enhancement helps users to correctly configure widgets and catch potential errors early in the configuration process, leading to a smoother widget setup experience.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-added-types-to-widget-configuration.md]:
```suggestion
To provide more clarity and improve validation when configuring widgets, type information has been added to the widget configuration. Previously, the widget configuration did not include any type information, which could lead to confusion and misconfiguration. With this change, each configuration property now specifies its expected type, such as string, number, or Boolean. 
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@janhommes @rahultsag Please approve

**Review by janhommes — APPROVED** [review]

**Pull request merged**